src/openrct2/ride/Vehicle.h

```diff
@@ -321,6 +321,11 @@ struct Vehicle : SpriteBase
     Vehicle* TrainHead() const;
     Vehicle* TrainTail() const;
 
+    uint16_t TrackType() const
```

Comment:
In general getters in the code base would have `Get` as a prefix. Could you change it to add the prefix.